Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -82,6 +82,46 @@ An impact defines the relation between an incident and a component. A component
 
 To reflect this, each component and incident can have a list of impacts, stating the type of impact and a reference to the incident or component, it refers to.
 
+Furthermore a severity value is added to the impact when referencing an component, to gauge the impacts severity on the system.
+
+### Severity
+
+A severity contains a name, that MUST be unique and WILL be used as identifier. The value marks the upper boundary of the severity.
+
+The severity value range is calculated by the previous severitys (SeverityA) value + 1 to the current severitys (SeverityB) value. These limits are inclusive.
+
+`SeverityA.value + 1 <=Severity <=SeverityB.value`
```

Comment:
This is a bit unclear at first glance because we have `SeverityA`, `SeverityB` and an additional `Severity` which stands for "all valid values falling into the range of `SeverityB`"without clearly stating so. Would using a range be clearer?

S<sub>B</sub>.severity ∈ (S<sub>A</sub>.value, S<sub>B</sub>.value)

Alternatively, it might be enough to just replace the un-annotated `Severity` in the middle with a clearer `severity range of SeverityB` or `SeverityB.severity`.

Reply:
Just noticed that my range is wrong, it should be [S<sub>A</sub>.value, S<sub>B</sub>.value), as the value of S<sub>A</sub> is not part of it.

Still, the following notation might be simpler and preferable:

``0, ... , SeverityA.value, SeverityA.value, + 1, ... , SeverityB.value - 1, SeverityB.value, SeverityB.value + 1, ... , 100
                        |<------------range of severity values for SeverityB------------->|
``